flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendRunTest.java

```diff
@@ -208,6 +209,31 @@ public void testParallelismWithOverflow() throws Exception {
         testFrontend.run(parameters);
     }
 
+    @Test(expected = CliArgsException.class)
+    public void testParallelismWithInvalidValue() throws Exception {
```

Comment:
maybe "testParallelismWithInvalidValue" -> "testInvalidNegativeParallelismOption"
because there is already a "testInvalidParallelismOption" test.